Library/Formula/camlp5.rb

```diff
@@ -13,20 +16,36 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-tex", "Install the pdf, ps, and tex documentation"
 
   depends_on "ocaml"
+  depends_on "gnu-sed" => :build if build.with?("doc") || build.with?("tex")
+  depends_on :tex => [:build, :optional]
+  depends_on "ghostscript" => :build if build.with?("tex")
 
   def install
-    if build.with? "strict"
-      strictness = "-strict"
-    else
-      strictness = "-transitional"
-    end
+    args = ["--prefix", prefix, "--mandir", man]
+    args << "--transitional" if build.without? "strict"
 
-    system "./configure", "-prefix", prefix, "-mandir", man, strictness
-    # this build fails if jobs are parallelized
-    ENV.deparallelize
+    system "./configure", *args
     system "make", "world.opt"
     system "make", "install"
+
+    if build.with?("doc") || build.with?("tex")
+      ENV.prepend_path "PATH", Formula["gnu-sed"].opt_libexec/"gnubin"
```

Comment:
The problem can be neatly summarised by Homebrew's desire to give users choice, even when it makes life difficult for us :smile:. Things that dupe or "enhance" system functionality with the same executable names are nearly universally `keg_only`, keep the same executable names, and `superenv` handles the situation smoothly when they are required.

However, `gnu-sed` like the `coreutils` formulae and a few others instead were prepended with the letter `g` by default because users wanted them in the `$PATH` without having to modify the `$PATH` themselves locally. Theoretically, it's a good argument. Realistically, it means where build scripts aren't checking for the alternative names we have to dance around the issue manually.